index.js

```diff
@@ -0,0 +1 @@
+module.exports = simpleTest => console.log(simpleTest);
```

Comment:
Hi, your library is very sample. But it can be useful if you add more features. Like the library called **debug**.
So, the recommendation is that you can add the label to define what kind of message you want to console.log and what file is triggering this console.log. Good luck!

Reply:
Thanks Amber, good idea! My goal was simply to finish, ha.